### PR TITLE
DES: Added DESController.java

### PR DESCRIPTION
**Pull request by PuravUdayDesai**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Number: #1553

### CODE REVIEW

**Comment by PuravUdayDesai** [comment]
@HarshCasper please review the pull request once, if any changes/improvements/modifications needed please let me know about the respective.

Sequence Number: 01

Issue Number: #1553 
Pull Request Number:  #2078

File: Java/cryptography/des/DESController.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	 * 		<li>DESCryptographyLogic.decrypt</li>
+	 * </ul>
+	 * 
+	 * And uses the following objects of types:
+	 * <ul>
+	 * 		<li>DESCryptographyLogic</li>
+	 * </ul>
+	 * 
+	 * @param args not used
+	 */
```
Comment (HarshCasper) [inline Java/cryptography/des/DESController.java]: What is the purpose of using these tags here?

File: Java/cryptography/des/DESController.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+			// Used for styling the output
+			if (i % 8 == 0 && i != 0)
+			{
+				System.out.print(" ");
+			}
+			System.out.print(decryptedText[i]);
+		}
+		
+	}// End of main(String[]) method
+}// End of class DESController
```
Comment (HarshCasper) [inline Java/cryptography/des/DESController.java]: Redundant Comments

File: Java/cryptography/des/DESController.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+		//setting original key
+		desCryptographyModels.setKey(key);
+		
+		//printing plain text
+		System.out.println("Plain Text:");
+		for (int i = 0; i < plainText.length; i++)
+		{
+			//used for styling the output
+			if (i % 8 == 0 && i != 0)
+			{
```
Comment (HarshCasper) [inline Java/cryptography/des/DESController.java]: Can the Core Logic of the Code shift into some other function? We should ideally use the main() function only for the Driver Code

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please make apt README changes as well

**Comment by PuravUdayDesai** [comment]
> Please make apt README changes as well

@HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.

**Comment by HarshCasper** [comment]
> > Please make apt README changes as well
> 
> @HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.

I have already added my comments. Please resolve them to continue forward. Also take a look at the merged PRs and have an insight into the entire workflow!

**Comment by PuravUdayDesai** [comment]
> > > Please make apt README changes as well
> > 
> > 
> > @HarshCasper Please elaborate the incorrect things you observed in the pull request and the changes you're looking forward.
> 
> I have already added my comments. Please resolve them to continue forward. Also take a look at the merged PRs and have an insight into the entire workflow!

Ok @HarshCasper Thank you for your comments, I'll update them accordingly and revert

File: Java/cryptography/des/DESController.java (hunk 1 above)
Comment (PuravUdayDesai) [reply Java/cryptography/des/DESController.java]: It is used maintain a javadoc, it just tells which all methods and objects are used in main method, if anything incorrect, please do let me now.

File: Java/cryptography/des/DESController.java (hunk 2 above)
Comment (PuravUdayDesai) [reply Java/cryptography/des/DESController.java]: What shall I do for it please guide

File: Java/cryptography/des/DESController.java (hunk 3 above)
Comment (PuravUdayDesai) [reply Java/cryptography/des/DESController.java]: Okay I understand your concern I'll do that for sure, really thank you for guidance.

File: Java/cryptography/des/DESController.java (hunk 1 above)
Comment (HarshCasper) [reply Java/cryptography/des/DESController.java]: Please remove them up. We are not maintaining a Javadoc here

File: Java/cryptography/des/DESController.java (hunk 2 above)
Comment (HarshCasper) [reply Java/cryptography/des/DESController.java]: Remove them up

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Java/cryptography/des/DESController.java (hunk 1 above)
Comment (PuravUdayDesai) [reply Java/cryptography/des/DESController.java]: Okay @HarshCasper I'll do that

File: Java/cryptography/des/DESController.java (hunk 2 above)
Comment (PuravUdayDesai) [reply Java/cryptography/des/DESController.java]: Ok @HarshCasper I'll do that

**Comment by PuravUdayDesai** [comment]
@HarshCasper I have made the told changes in each and every file in which I observed the given issues.

**Comment by PuravUdayDesai** [comment]
@atarax665 please reopen the closed issue, I have completed the given changes, sorry for any delay.

**Comment by HarshCasper** [comment]
Closing this Issue because of no updates.

**Comment by HarshCasper** [comment]
Closing this PR because of no updates.

**Comment by PuravUdayDesai** [comment]
@HarshCasper I had submitted the updates which you requested 12 days ago, for which other updates you are looking for?

**Pull request closed without merging**